easybuild/easyconfigs/m/matplotlib/matplotlib-2.1.0-foss-2017b-Python-3.6.2.eb

```diff
@@ -0,0 +1,52 @@
+easyblock = 'Bundle'
+
+name = 'matplotlib'
+version = '2.1.0'
+versionsuffix = '-Python-%(pyver)s'
+
+libpng_ver = '1.6.32'
+
+homepage = 'http://matplotlib.org'
+description = """matplotlib is a python 2D plotting library which produces publication quality figures in a variety of
+ hardcopy formats and interactive environments across platforms. matplotlib can be used in python scripts, the python
+ and ipython shell, web application servers, and six graphical user interface toolkits."""
+
+toolchain = {'name': 'foss', 'version': '2017b'}
+
+# this is a bundle of Python packages
+exts_defaultclass = 'PythonPackage'
+
+dependencies = [
+    ('Python', '3.6.2'),
+    ('libpng', libpng_ver),
+    ('freetype', '2.7.1', '-libpng-%s' % libpng_ver, ('GCCcore', '6.4.0')),
+]
+
+exts_list = [
+    ('Cycler', '0.10.0', {
+        'modulename': 'cycler',
+        'source_tmpl': 'cycler-%(version)s.tar.gz',
+        'source_urls': ['https://pypi.python.org/packages/source/C/Cycler'],
+        'checksums': [
+            'cd7b2d1018258d7247a71425e9f26463dfb444d411c39569972f4ce586b0c9d8',  # cycler-0.10.0.tar.gz
+        ],
+    }),
+    ('matplotlib', '2.1.0', {
```

Comment:
please use `name` and `version` here; this is an issue with `--inject-checksums` I need to fix...